src/DotLiquid/Util/CharEnumerator.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DotLiquid.Util
+{
+    internal class CharEnumerator : IEnumerator<char>
+    {
+        private readonly String str;
+        private int index;
+
+        internal CharEnumerator(String str)
```

Comment:
String already implements `IEnumerable<char>` so you can call GetEnumerator and get the `IEnumerator<char>`. This class appears unnecessary.

Reply:
Two reasons for this:
1. NET Standard 1.3 doesn't support it. Hence my attempt at bumping which was shot down
2. The .NET version is sealed and marks key properties as private. With our version, we exposed Position and Remaining, as well as Next and Previous characters so we can 'Peek' more like a stream.

Reply:
oh, ok, makes sense then.